packit/distgit.py

```diff
@@ -173,6 +172,11 @@ def downstream_config(self) -> Optional[PackageConfig]:
                 return None
         return self._downstream_config
 
+    @property
+    def pkg_tool(self) -> str:
+        """Returns the packaging tool. Prefers the package-level override."""
+        return self.package_config.pkg_tool or self.config.pkg_tool
```

Comment:
Don't you think `package-level` might be an overloaded term here?

Reply:
I'm not sure I understand :D Do you mean the confusion between package as in _packaging tool_ vs _Packit config_?